frontend/server/src/main/java/com/amazonaws/ml/mms/util/ConfigManager.java

```diff
@@ -445,6 +453,33 @@ private File findMmsHome() {
         return cwd;
     }
 
+    private void enableAsyncLogging() {
+        enableAsyncLogging(Logger.getRootLogger());
+        enableAsyncLogging(Logger.getLogger(MODEL_METRICS_LOGGER));
+        enableAsyncLogging(Logger.getLogger(MODEL_LOGGER));
+        enableAsyncLogging(Logger.getLogger(MMS_METRICS_LOGGER));
+        enableAsyncLogging(Logger.getLogger("ACCESS_LOG"));
+        enableAsyncLogging(Logger.getLogger("com.amazonaws.ml.mms"));
+    }
+
+    private void enableAsyncLogging(Logger logger) {
+        AsyncAppender asyncAppender = new AsyncAppender();
+
+        @SuppressWarnings("unchecked")
```

Comment:
Why is the warning suppressed here ?

Reply:
Warning is caused by log4j API, logger.getAllAppenders() didn't return parameterized Enumeration. We have to suppress it here.